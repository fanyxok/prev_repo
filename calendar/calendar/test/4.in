Shanghaitech 776370 Sem 16
G pass_year 435326
G pass_month 3619360
S pass_day 17649866
S print_month
G pass_year -552890
S pass_day 45068551
S pass_year -466409
G print_month
S print_today
G print_today
S print_month
G print_month
S print_year
G print_year
