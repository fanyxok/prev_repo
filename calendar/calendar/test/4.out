Spst                                
┌────┬────┬────┬────┬────┬────┬────┐
│ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│
├────┼────┼────┼────┼────┼────┼────┤
│    │    │    │    │    │ 01 │ 02 │
├────┼────┼────┼────┼────┼────┼────┤
│ 03 │ 04 │ 05 │ 06 │ 07 │ 08 │ 09 │
├────┼────┼────┼────┼────┼────┼────┤
│ 0A │ 0B │ 0C │ 0D │ 0E │ 0F │ 10 │
├────┼────┼────┼────┼────┼────┼────┤
│ 11 │ 12 │ 13 │ 14 │ 15 │ 16 │ 17 │
├────┼────┼────┼────┼────┼────┼────┤
│ 18 │ 19 │ 1A │ 1B │ 1C │ 1D │ 1E │
├────┼────┼────┼────┼────┼────┼────┤
│ 1F │ 20 │ 21 │ 22 │ 23 │ 24 │ 25 │
├────┼────┼────┼────┼────┼────┼────┤
│ 26 │ 27 │ 28 │    │    │    │    │
└────┴────┴────┴────┴────┴────┴────┘
Aug                                 
┌────┬────┬────┬────┬────┬────┬────┐
│ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│
├────┼────┼────┼────┼────┼────┼────┤
│    │    │    │    │    │    │ 01 │
├────┼────┼────┼────┼────┼────┼────┤
│ 02 │ 03 │ 04 │ 05 │ 06 │ 07 │ 08 │
├────┼────┼────┼────┼────┼────┼────┤
│ 09 │ 10 │ 11 │ 12 │ 13 │ 14 │ 15 │
├────┼────┼────┼────┼────┼────┼────┤
│ 16 │ 17 │ 18 │ 19 │ 20 │ 21 │ 22 │
├────┼────┼────┼────┼────┼────┼────┤
│ 23 │ 24 │ 25 │ 26 │ 27 │ 28 │ 29 │
├────┼────┼────┼────┼────┼────┼────┤
│ 30 │ 31 │    │    │    │    │    │
└────┴────┴────┴────┴────┴────┴────┘
395022 Spst 3
395040 Aug 3
Spst                                
┌────┬────┬────┬────┬────┬────┬────┐
│ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│
├────┼────┼────┼────┼────┼────┼────┤
│    │    │    │    │    │    │ 01 │
├────┼────┼────┼────┼────┼────┼────┤
│ 02 │ 03 │ 04 │ 05 │ 06 │ 07 │ 08 │
├────┼────┼────┼────┼────┼────┼────┤
│ 09 │ 0A │ 0B │ 0C │ 0D │ 0E │ 0F │
├────┼────┼────┼────┼────┼────┼────┤
│ 10 │ 11 │ 12 │ 13 │ 14 │ 15 │ 16 │
├────┼────┼────┼────┼────┼────┼────┤
│ 17 │ 18 │ 19 │ 1A │ 1B │ 1C │ 1D │
├────┼────┼────┼────┼────┼────┼────┤
│ 1E │ 1F │ 20 │ 21 │ 22 │ 23 │ 24 │
├────┼────┼────┼────┼────┼────┼────┤
│ 25 │ 26 │ 27 │ 28 │    │    │    │
└────┴────┴────┴────┴────┴────┴────┘
Aug                                 
┌────┬────┬────┬────┬────┬────┬────┐
│ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│
├────┼────┼────┼────┼────┼────┼────┤
│    │    │    │    │    │    │ 01 │
├────┼────┼────┼────┼────┼────┼────┤
│ 02 │ 03 │ 04 │ 05 │ 06 │ 07 │ 08 │
├────┼────┼────┼────┼────┼────┼────┤
│ 09 │ 10 │ 11 │ 12 │ 13 │ 14 │ 15 │
├────┼────┼────┼────┼────┼────┼────┤
│ 16 │ 17 │ 18 │ 19 │ 20 │ 21 │ 22 │
├────┼────┼────┼────┼────┼────┼────┤
│ 23 │ 24 │ 25 │ 26 │ 27 │ 28 │ 29 │
├────┼────┼────┼────┼────┼────┼────┤
│ 30 │ 31 │    │    │    │    │    │
└────┴────┴────┴────┴────┴────┴────┘
Sist                                 Spst                                 Slst                                
┌────┬────┬────┬────┬────┬────┬────┐ ┌────┬────┬────┬────┬────┬────┬────┐ ┌────┬────┬────┬────┬────┬────┬────┐
│ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│ │ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│ │ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 01 │ 02 │ 03 │ 04 │ 05 │ 06 │ 07 │ │    │    │    │    │    │    │ 01 │ │    │    │    │    │ 01 │ 02 │ 03 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 08 │ 09 │ 0A │ 0B │ 0C │ 0D │ 0E │ │ 02 │ 03 │ 04 │ 05 │ 06 │ 07 │ 08 │ │ 04 │ 05 │ 06 │ 07 │ 08 │ 09 │ 0A │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 0F │ 10 │ 11 │ 12 │ 13 │ 14 │ 15 │ │ 09 │ 0A │ 0B │ 0C │ 0D │ 0E │ 0F │ │ 0B │ 0C │ 0D │ 0E │ 0F │ 10 │ 11 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 16 │ 17 │ 18 │ 19 │ 1A │ 1B │ 1C │ │ 10 │ 11 │ 12 │ 13 │ 14 │ 15 │ 16 │ │ 12 │ 13 │ 14 │ 15 │ 16 │ 17 │ 18 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 1D │ 1E │ 1F │ 20 │ 21 │ 22 │ 23 │ │ 17 │ 18 │ 19 │ 1A │ 1B │ 1C │ 1D │ │ 19 │ 1A │ 1B │ 1C │ 1D │ 1E │ 1F │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 24 │ 25 │ 26 │ 27 │ 28 │ 29 │    │ │ 1E │ 1F │ 20 │ 21 │ 22 │ 23 │ 24 │ │ 20 │ 21 │ 22 │ 23 │ 24 │ 25 │ 26 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│    │    │    │    │    │    │    │ │ 25 │ 26 │ 27 │ 28 │    │    │    │ │ 27 │    │    │    │    │    │    │
└────┴────┴────┴────┴────┴────┴────┘ └────┴────┴────┴────┴────┴────┴────┘ └────┴────┴────┴────┴────┴────┴────┘
Sem                                  Sca                                  Ims                                 
┌────┬────┬────┬────┬────┬────┬────┐ ┌────┬────┬────┬────┬────┬────┬────┐ ┌────┬────┬────┬────┬────┬────┬────┐
│ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│ │ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│ │ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│    │ 01 │ 02 │ 03 │ 04 │ 05 │ 06 │ │ 01 │ 02 │ 03 │ 04 │ 05 │ 06 │ 07 │ │    │    │    │    │    │ 01 │ 02 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 07 │ 08 │ 09 │ 0A │ 0B │ 0C │ 0D │ │ 08 │ 09 │ 0A │ 0B │ 0C │ 0D │ 0E │ │ 03 │ 04 │ 05 │ 06 │ 07 │ 08 │ 09 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 0E │ 0F │ 10 │ 11 │ 12 │ 13 │ 14 │ │ 0F │ 10 │ 11 │ 12 │ 13 │ 14 │ 15 │ │ 0A │ 0B │ 0C │ 0D │ 0E │ 0F │ 10 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 15 │ 16 │ 17 │ 18 │ 19 │ 1A │ 1B │ │ 16 │ 17 │ 18 │ 19 │ 1A │ 1B │ 1C │ │ 11 │ 12 │ 13 │ 14 │ 15 │ 16 │ 17 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 1C │ 1D │ 1E │ 1F │ 20 │ 21 │ 22 │ │ 1D │ 1E │ 1F │ 20 │ 21 │ 22 │ 23 │ │ 18 │ 19 │ 1A │ 1B │ 1C │ 1D │ 1E │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 23 │ 24 │ 25 │ 26 │ 27 │ 28 │ 29 │ │ 24 │ 25 │ 26 │ 27 │ 28 │    │    │ │ 1F │ 20 │ 21 │ 22 │ 23 │ 24 │ 25 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│    │    │    │    │    │    │    │ │    │    │    │    │    │    │    │ │ 26 │ 27 │    │    │    │    │    │
└────┴────┴────┴────┴────┴────┴────┘ └────┴────┴────┴────┴────┴────┴────┘ └────┴────┴────┴────┴────┴────┴────┘
Ihuman                               Siais                                Ih                                  
┌────┬────┬────┬────┬────┬────┬────┐ ┌────┬────┬────┬────┬────┬────┬────┐ ┌────┬────┬────┬────┬────┬────┬────┐
│ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│ │ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│ │ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│    │    │ 01 │ 02 │ 03 │ 04 │ 05 │ │    │ 01 │ 02 │ 03 │ 04 │ 05 │ 06 │ │    │    │    │    │    │    │ 01 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 06 │ 07 │ 08 │ 09 │ 0A │ 0B │ 0C │ │ 07 │ 08 │ 09 │ 0A │ 0B │ 0C │ 0D │ │ 02 │ 03 │ 04 │ 05 │ 06 │ 07 │ 08 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 0D │ 0E │ 0F │ 10 │ 11 │ 12 │ 13 │ │ 0E │ 0F │ 10 │ 11 │ 12 │ 13 │ 14 │ │ 09 │ 0A │ 0B │ 0C │ 0D │ 0E │ 0F │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 14 │ 15 │ 16 │ 17 │ 18 │ 19 │ 1A │ │ 15 │ 16 │ 17 │ 18 │ 19 │ 1A │ 1B │ │ 10 │ 11 │ 12 │ 13 │ 14 │ 15 │ 16 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 1B │ 1C │ 1D │ 1E │ 1F │ 20 │ 21 │ │ 1C │ 1D │ 1E │ 1F │ 20 │ 21 │ 22 │ │ 17 │ 18 │ 19 │ 1A │ 1B │ 1C │ 1D │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 22 │ 23 │ 24 │ 25 │ 26 │ 27 │ 28 │ │ 23 │ 24 │ 25 │ 26 │ 27 │ 28 │    │ │ 1E │ 1F │ 20 │ 21 │ 22 │ 23 │ 24 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 29 │    │    │    │    │    │    │ │    │    │    │    │    │    │    │ │ 25 │ 26 │ 27 │    │    │    │    │
└────┴────┴────┴────┴────┴────┴────┘ └────┴────┴────┴────┴────┴────┴────┘ └────┴────┴────┴────┴────┴────┴────┘
Jan                                  Feb                                  Mar                                 
┌────┬────┬────┬────┬────┬────┬────┐ ┌────┬────┬────┬────┬────┬────┬────┐ ┌────┬────┬────┬────┬────┬────┬────┐
│ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│ │ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│ │ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│    │    │    │ 01 │ 02 │ 03 │ 04 │ │    │    │    │    │    │    │ 01 │ │ 01 │ 02 │ 03 │ 04 │ 05 │ 06 │ 07 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 05 │ 06 │ 07 │ 08 │ 09 │ 10 │ 11 │ │ 02 │ 03 │ 04 │ 05 │ 06 │ 07 │ 08 │ │ 08 │ 09 │ 10 │ 11 │ 12 │ 13 │ 14 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 12 │ 13 │ 14 │ 15 │ 16 │ 17 │ 18 │ │ 09 │ 10 │ 11 │ 12 │ 13 │ 14 │ 15 │ │ 15 │ 16 │ 17 │ 18 │ 19 │ 20 │ 21 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 19 │ 20 │ 21 │ 22 │ 23 │ 24 │ 25 │ │ 16 │ 17 │ 18 │ 19 │ 20 │ 21 │ 22 │ │ 22 │ 23 │ 24 │ 25 │ 26 │ 27 │ 28 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 26 │ 27 │ 28 │ 29 │ 30 │ 31 │    │ │ 23 │ 24 │ 25 │ 26 │ 27 │ 28 │ 29 │ │ 29 │ 30 │ 31 │    │    │    │    │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│    │    │    │    │    │    │    │ │    │    │    │    │    │    │    │ │    │    │    │    │    │    │    │
└────┴────┴────┴────┴────┴────┴────┘ └────┴────┴────┴────┴────┴────┴────┘ └────┴────┴────┴────┴────┴────┴────┘
Apr                                  May                                  Jun                                 
┌────┬────┬────┬────┬────┬────┬────┐ ┌────┬────┬────┬────┬────┬────┬────┐ ┌────┬────┬────┬────┬────┬────┬────┐
│ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│ │ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│ │ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│    │    │    │ 01 │ 02 │ 03 │ 04 │ │    │    │    │    │    │ 01 │ 02 │ │    │ 01 │ 02 │ 03 │ 04 │ 05 │ 06 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 05 │ 06 │ 07 │ 08 │ 09 │ 10 │ 11 │ │ 03 │ 04 │ 05 │ 06 │ 07 │ 08 │ 09 │ │ 07 │ 08 │ 09 │ 10 │ 11 │ 12 │ 13 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 12 │ 13 │ 14 │ 15 │ 16 │ 17 │ 18 │ │ 10 │ 11 │ 12 │ 13 │ 14 │ 15 │ 16 │ │ 14 │ 15 │ 16 │ 17 │ 18 │ 19 │ 20 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 19 │ 20 │ 21 │ 22 │ 23 │ 24 │ 25 │ │ 17 │ 18 │ 19 │ 20 │ 21 │ 22 │ 23 │ │ 21 │ 22 │ 23 │ 24 │ 25 │ 26 │ 27 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 26 │ 27 │ 28 │ 29 │ 30 │    │    │ │ 24 │ 25 │ 26 │ 27 │ 28 │ 29 │ 30 │ │ 28 │ 29 │ 30 │    │    │    │    │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│    │    │    │    │    │    │    │ │ 31 │    │    │    │    │    │    │ │    │    │    │    │    │    │    │
└────┴────┴────┴────┴────┴────┴────┘ └────┴────┴────┴────┴────┴────┴────┘ └────┴────┴────┴────┴────┴────┴────┘
Jul                                  Aug                                  Sep                                 
┌────┬────┬────┬────┬────┬────┬────┐ ┌────┬────┬────┬────┬────┬────┬────┐ ┌────┬────┬────┬────┬────┬────┬────┐
│ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│ │ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│ │ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│    │    │    │ 01 │ 02 │ 03 │ 04 │ │    │    │    │    │    │    │ 01 │ │    │    │ 01 │ 02 │ 03 │ 04 │ 05 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 05 │ 06 │ 07 │ 08 │ 09 │ 10 │ 11 │ │ 02 │ 03 │ 04 │ 05 │ 06 │ 07 │ 08 │ │ 06 │ 07 │ 08 │ 09 │ 10 │ 11 │ 12 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 12 │ 13 │ 14 │ 15 │ 16 │ 17 │ 18 │ │ 09 │ 10 │ 11 │ 12 │ 13 │ 14 │ 15 │ │ 13 │ 14 │ 15 │ 16 │ 17 │ 18 │ 19 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 19 │ 20 │ 21 │ 22 │ 23 │ 24 │ 25 │ │ 16 │ 17 │ 18 │ 19 │ 20 │ 21 │ 22 │ │ 20 │ 21 │ 22 │ 23 │ 24 │ 25 │ 26 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 26 │ 27 │ 28 │ 29 │ 30 │ 31 │    │ │ 23 │ 24 │ 25 │ 26 │ 27 │ 28 │ 29 │ │ 27 │ 28 │ 29 │ 30 │    │    │    │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│    │    │    │    │    │    │    │ │ 30 │ 31 │    │    │    │    │    │ │    │    │    │    │    │    │    │
└────┴────┴────┴────┴────┴────┴────┘ └────┴────┴────┴────┴────┴────┴────┘ └────┴────┴────┴────┴────┴────┴────┘
Oct                                  Nov                                  Dec                                 
┌────┬────┬────┬────┬────┬────┬────┐ ┌────┬────┬────┬────┬────┬────┬────┐ ┌────┬────┬────┬────┬────┬────┬────┐
│ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│ │ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│ │ Sun│ Mon│Tues│ Wed│Thur│ Fri│ Sat│
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│    │    │    │    │ 01 │ 02 │ 03 │ │ 01 │ 02 │ 03 │ 04 │ 05 │ 06 │ 07 │ │    │    │ 01 │ 02 │ 03 │ 04 │ 05 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 04 │ 05 │ 06 │ 07 │ 08 │ 09 │ 10 │ │ 08 │ 09 │ 10 │ 11 │ 12 │ 13 │ 14 │ │ 06 │ 07 │ 08 │ 09 │ 10 │ 11 │ 12 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 11 │ 12 │ 13 │ 14 │ 15 │ 16 │ 17 │ │ 15 │ 16 │ 17 │ 18 │ 19 │ 20 │ 21 │ │ 13 │ 14 │ 15 │ 16 │ 17 │ 18 │ 19 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 18 │ 19 │ 20 │ 21 │ 22 │ 23 │ 24 │ │ 22 │ 23 │ 24 │ 25 │ 26 │ 27 │ 28 │ │ 20 │ 21 │ 22 │ 23 │ 24 │ 25 │ 26 │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│ 25 │ 26 │ 27 │ 28 │ 29 │ 30 │ 31 │ │ 29 │ 30 │    │    │    │    │    │ │ 27 │ 28 │ 29 │ 30 │ 31 │    │    │
├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤ ├────┼────┼────┼────┼────┼────┼────┤
│    │    │    │    │    │    │    │ │    │    │    │    │    │    │    │ │    │    │    │    │    │    │    │
└────┴────┴────┴────┴────┴────┴────┘ └────┴────┴────┴────┴────┴────┴────┘ └────┴────┴────┴────┴────┴────┴────┘
